mpas_analysis/ocean/meridional_overturning_circulation.py

```diff
@@ -410,16 +439,50 @@ def _compute_moc_time_series_postprocess(config, runStreams, variableMap,
                                          dictRegion):  # {{{
     '''compute MOC time series as a post-process'''
 
-    def write_file(dsMOCTimeSeries):
-        # make sure the Time dimension is in ascending order by Time
-        indices = dsMOCTimeSeries.Time.argsort()
-        dsMOCTimeSeries = dsMOCTimeSeries.isel(Time=indices)
-        dsMOCTimeSeries.Time.attrs['units'] = 'days since 0001-01-01'
-        dsMOCTimeSeries.mocAtlantic26.attrs['units'] = 'Sv (10^6 m^3/s)'
-        dsMOCTimeSeries.mocAtlantic26.attrs['description'] = \
-            'Max MOC Atlantic streamfunction nearest to RAPID Array latitude (26.5N)'
-        dsMOCTimeSeries.to_netcdf(outputFileTseries)
-        return dsMOCTimeSeries
+    def compute_moc_time_series_part(timeIndices, firstCall):
```

Comment:
I was hoping to avoid using `partial` and to have to pass a *ton* of arguments.  I will switch to having this as a separate, private function but in my opinion this doesn't make the code clearer.